cmake/Modules/FindFAUST.cmake

```diff
@@ -0,0 +1,19 @@
+# Find Faust2
+
+find_path(
+  FAUST_INCLUDE_DIR faust/dsp/llvm-dsp.h
+  HINTS
+  /opt/lib/faust/architecture/
+  /usr/lib/faust/architecture/
+  /usr/local/lib/faust/architecture/
+  "${FAUST_INCLUDE_DIR_HINT}"
+  )
+
+set(FAUST_NAMES ${FAUST_NAMES} libfaust.so libfaust.dylib faust.dll faust libfaust)
+find_library(FAUST_LIBRARY
+  NAMES ${FAUST_NAMES}
+  HINTS "${FAUST_LIB_DIR_HINT}")
+
+if(FAUST_INCLUDE_DIR AND FAUST_LIBRARY)
```

Comment:
You can use `find_package_handle_standard_args(FAUST FAUST_INCLUDE_DIR FAUST_LIBRARY)` to set this. This macro also handles REQUIRED and QUIETLY keywords. You need to `include(FindPackageHandleStandardArgs)` to use it though.

Reply:
@hlolli did you make these changes?

Reply:
No not these, I'll add them!

Reply:
Done!

Reply:
all ready to merge?